spec/factories/books_attributes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  trait :books_attributes do
```

Comment:
i dont understand why a whole factory is only a trait 🤔

Reply:
I've used this trait to share common attributes between books and book suggestions